Userland/DevTools/Playground/main.cpp

```diff
@@ -104,6 +104,35 @@ int main(int argc, char** argv)
     editor.set_should_autocomplete_automatically(true);
     editor.set_automatic_indentation_enabled(true);
 
+    RefPtr<GUI::Action> m_save_as_action = GUI::CommonActions::make_save_as_action([&](auto&) {
+        Optional<String> save_path = GUI::FilePicker::get_save_filepath(window, "Untitled", "gml");
+        if (!save_path.has_value())
+            return;
+
+        if (!editor.write_to_file(save_path.value())) {
+            GUI::MessageBox::show(window, "Unable to save file.\n", "Error", GUI::MessageBox::Type::Error);
+            return;
+        }
```

Comment:
There's nothing after this, so no return is needed.

```suggestion
        if (!editor.write_to_file(save_path.value()))
            GUI::MessageBox::show(window, "Unable to save file.\n", "Error", GUI::MessageBox::Type::Error);
```